tools/targets/__init__.py

```diff
@@ -26,17 +26,13 @@
 from collections import namedtuple
 from copy import copy
 from future.utils import raise_from
+from os.path import dirname, abspath, join
```

Comment:
Does any code in this file use `dirname`, `abspath`, or `join`? If not, this is probably best added to `ARM_MUSCA.py` instead, where those are used.

Reply:
They are still used in this file